src/snowflake/snowpark/session.py

```diff
@@ -834,16 +857,17 @@ def add_requirements(self, file_path: str) -> None:
         with open(file_path) as f:
             for line in f:
                 package = line.rstrip()
-                if package:
+                if package and len(package) > 0:
```

Comment:
is there a case when `if package` is `True` but `len(package) > 0` is `False` when `package` is a `str`

Reply:
True, this is redundant; removed